internal/interpreter/api.rs

```diff
@@ -83,6 +83,26 @@ impl From<LangType> for ValueType {
     }
 }
 
+#[cfg(feature = "json")]
+fn color_to_string(color: &i_slint_core::Color) -> String {
+    let encoded = color.as_argb_encoded();
+
+    let a = ((encoded & 0xff000000) >> 24) as u8;
```

Comment:
Maybe convert to RGB u8 so we don't rely on the internal encoding here

Reply:
Done.